PMSCore/UserInterface.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Any particular reason for all this includes? Could we clean up this a bit?

Reply:
Forgot to mention you can call IUser instead of UserInterface
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

Reply:
Removed on latest push.